File: code-contests-dataset/solutions/qwen3-cpp-dpo/ccc06j1.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    // Define calorie values for each menu item
    vector<int> burger_calories = {461, 431, 420, 0};     // 1-4 burger choices
    vector<int> side_calories = {100, 57, 70, 0};         // 1-4 side choices
    vector<int> drink_calories = {130, 160, 118, 0};      // 1-4 drink choices
    vector<int> dessert_calories = {167, 266, 75, 0};     // 1-4 dessert choices
    
    int burger_choice, side_choice, drink_choice, dessert_choice;
    
    // Read choices
    cin >> burger_choice >> side_choice >> drink_choice >> dessert_choice;
    
    // Calculate total calories
    int total_calories = burger_calories[burger_choice - 1] + 
                        side_calories[side_choice - 1] + 
                        drink_calories[drink_choice - 1] + 
                        dessert_calories[dessert_choice - 1];
    
    cout << "Your total Calorie count is " << total_calories << "." << endl;
    
    return 0;
}

File: code-contests-dataset/solutions/qwen3-rust-dpo/ccc06j1.rs
use std::io::{self, Read};
use rayon::prelude::*;

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    
    let mut lines = input.lines();
    let burger_choice = lines.next().unwrap().parse::<i32>().unwrap();
    let side_choice = lines.next().unwrap().parse::<i32>().unwrap();
    let drink_choice = lines.next().unwrap().parse::<i32>().unwrap();
    let dessert_choice = lines.next().unwrap().parse::<i32>().unwrap();
    
    let burger_calories = [0, 461, 431, 420, 0];
    let side_calories = [0, 100, 57, 70, 0];
    let drink_calories = [0, 130, 160, 118, 0];
    let dessert_calories = [0, 167, 266, 75, 0];
    
    let total = burger_calories[burger_choice as usize] +
                side_calories[side_choice as usize] +
                drink_calories[drink_choice as usize] +
                dessert_calories[dessert_choice as usize];
                
    println!("Your total Calorie count is {}.", total);
}